libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -426,3 +437,52 @@ queryStakePoolDefaultVote ::
 queryStakePoolDefaultVote nes poolId =
   defaultStakePoolVote poolId (nes ^. nesEsL . epochStateStakePoolsL) $
     nes ^. nesEsL . esLStateL . lsCertStateL . certDStateL . accountsL
+
+-- | Used only for the `queryPoolState` query. This resembles the older way of
+-- representing StakePoolState in Ledger.
+data PState' = PState'
+  { psStakePoolParams :: !(Map (KeyHash 'StakePool) PoolParams)
+  , psFutureStakePoolParams :: !(Map (KeyHash 'StakePool) PoolParams)
+  , psRetiring' :: !(Map (KeyHash 'StakePool) EpochNo)
+  , psDeposits :: !(Map (KeyHash 'StakePool) Coin)
+  }
```

Comment:
This is the worst naming that I've seen :smiling_face_with_tear: 
All these ticks make me wanna cry. How about adding a `query` prefix instead and adding `result` for disambiguation from the actual query types:
```suggestion
data QueryPoolStateResult = QueryPoolStateResult
  { qpsrStakePoolParams :: !(Map (KeyHash 'StakePool) PoolParams)
  , qpsrFutureStakePoolParams :: !(Map (KeyHash 'StakePool) PoolParams)
  , qpsrRetiring :: !(Map (KeyHash 'StakePool) EpochNo)
  , qpsrDeposits :: !(Map (KeyHash 'StakePool) Coin)
  }
```

Reply:
:slightly_smiling_face: yes I admit the naming was terrible. Thanks for the suggestion!